tests/CompleteCircuitBreakerTest.php

```diff
@@ -26,10 +26,12 @@ public function setUp() {
 	public function registerRequests(array $requests){
 		foreach ($requests as $time => $success) {
 			$this->timeProvider->set($time);
-			if ($success) {
+			if ($success==TRUE) {
 				$this->sut->registerSuccess();
-			}else{
+			}elseif($success==FALSE){
```

Comment:
Same here. If `$success` must be boolean `FALSE`, compare with `===`. Otherwise, we don't need the comparison.